keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -246,6 +251,23 @@ public class KeeperPlugin : Plugin<Project> {
     }
   }
 
+  private fun Project.androidEmbeddedJarRegularFileProvider(
+      appExtension: AppExtension,
+      path: String,
+      required: Boolean
+  ): Provider<RegularFile> =
+    layout.file(provider {
```

Comment:
Let's make this a normal function body since it's fairly long

Reply:
Not sure about the function size, but I guess the complexity is coming from the provider + file resolution in just 1 function.
So I split it in 2 and you can see the result in ca06e81